docs/weblog/README.md

```diff
@@ -126,6 +126,25 @@ In particular, it accepts and parse JSON and XML content. A typical XML content
     content
 </string>
 ```
+
+## \[GET, POST, OPTIONS\] /set_tag/%s/%d
+
+This endpoint must accept two required parameters (the first is a string, the second is an integer) and are part of the URL path. 
+
+This endpoint must accept all query parameters and all content types.
+
+The first path parameter must be written in the span with the tag `appsec.events.system_tests_appsec_event.value` and the parameter as value.
+
+The second path parameter must be used as a response status code.
+
+All query parameters (key, value) must be used as (key, value) in the response headers.
```

Comment:
Unclear to me. Maybe provide an example?

Reply:
Yes, good idea, I add a full example soon.